.github/ISSUE_TEMPLATE/mailchain_improvement_proposal.md

```diff
@@ -37,4 +37,4 @@ Test cases for an implementation are mandatory for MIPs that are affecting speci
 
 ## Related MIP
 
-Links to any related MIPs.
```

Comment:
Did you intend to change this?

Reply:
Unfortunately I did not, however, I had pre-commit installed and it seemed to have made some changes. Would you like me to revert them?

Reply:
No its good.